.circleci/config.yml

```diff
@@ -22,8 +22,7 @@ commands:
 
   node_setup:
     steps:
-      - node/install:
-          node-version: "" # use .nvmrc
+      - node/install
       - node/install-packages:
           include-branch-in-cache-key: false
```

Comment:
Actually it looks like we can remove `node_setup` completely.

Reply:
It's still referenced by the end to end tests and commitlint steps

Reply:
Oh, ok, didn't see it in the diff so assumed it was only needed by printnodeversion.